### PR TITLE
Building an aggregate loss curve per realization

### PR DESCRIPTION
**Pull request by micheles**
Closes https://github.com/gem/oq-risklib/issues/499. The demos are running here: https://ci.openquake.org/job/zdevel_oq-risklib/1174/

### CODE REVIEW

File: openquake/calculators/event_based_risk.py [hunk 1]
```diff
@@ -46,6 +82,17 @@ def square(L, R, factory):
     return losses
 
 
+def old_loss_curves(asset_values, rcurves, ordinal, ratios):
+    """
+    Build loss curves in the old format (i.e. (losses, poes)) from
+    loss curves in the new format (i.e. poes).
+    """
```
Comment (ptormene) [inline openquake/calculators/event_based_risk.py]: You could add the parameters description to the docstring

Comment (micheles) [inline openquake/calculators/event_based_risk.py]: Uhm... instead, I will add an underscore in front of this function, after all it is an internal detail that may even disappear in the future.

Comment (ptormene) [inline openquake/calculators/event_based_risk.py]: ok

**Comment by ptormene** [comment]
LGTM

**Pull request merged**